tests/end_to_end/helpers.h

```diff
@@ -673,14 +673,7 @@ static ebpf_program_data_t _ebpf_xdp_test_program_data = {
 
 // Bind.
 static ebpf_program_data_t _ebpf_bind_program_data = {
-    EBPF_PROGRAM_DATA_HEADER,
-    &_ebpf_bind_program_info,
-    NULL,
-    NULL,
-    NULL,
-    NULL,
-    0,
-    PROGRAM_DATA_CAPABILITY_CONTEXT_HEADER};
+    EBPF_PROGRAM_DATA_HEADER, &_ebpf_bind_program_info, NULL, NULL, NULL, NULL, 0, {true}};
```

Comment:
can you do 
`{.supports_context_header = true}`
here which makes it clear what the "true" means?  Same question in 907

Reply:
Fixed.